src/drivers/rc/sumd/module.yaml

```diff
@@ -0,0 +1,7 @@
+module_name: RC SUMD input driver
+serial_config:
+    - command: "rc_sumd start -d ${SERIAL_DEV}"
+      port_config_param:
+        name: RC_SUMD_PRT_CFG
+        group: Serial
+        default: RC
```

Comment:
Need to be default disabled. Only one can run at a time.

Reply:
Thanks, should be fixed now.